### DISCUSSION TITLE
can i ask how to build scenebuilder 8.5 for windows - native build

### DISCUSSION THREAD

**Discussion by icyhoty2k** [gluonhq/scenebuilder#510] [Q&A]
can i ask how to build scenebuilder 8.5 for windows - native build (exe file)

**Comment by Oliver-Loeffler** [comment]
I cannot answer this and will investigate on how this could work. For Java 8 there was the JavaFX packager. I remember this one was able to package SB to an exe.

**Reply by Oliver-Loeffler** [reply]
@icyhoty2k Is this still of interest?
